tests/Feature/Http/Controllers/Auth/PasswordConfirmationTest.php

```diff
@@ -41,7 +41,7 @@ public function testPasswordIsNotConfirmedWithInvalidPassword()
     {
         $this->markTestSkipped();
 
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````